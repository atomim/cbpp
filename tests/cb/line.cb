AddText "this is line #"+__LINE__
AddText "and this is " + __LINE__


AddText __LINE__+" jea" 
