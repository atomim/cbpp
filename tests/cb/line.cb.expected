AddText "this is line #"+1
AddText "and this is " + 2


AddText 5+" jea" 
